[2024-10-08T10:08:35.999+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-08T10:08:36.016+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_etl_01.run_extract_container_1 scheduled__2024-10-07T00:00:00+00:00 [queued]>
[2024-10-08T10:08:36.024+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_etl_01.run_extract_container_1 scheduled__2024-10-07T00:00:00+00:00 [queued]>
[2024-10-08T10:08:36.024+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-08T10:08:36.036+0000] {taskinstance.py:2888} INFO - Executing <Task(DockerOperator): run_extract_container_1> on 2024-10-07 00:00:00+00:00
[2024-10-08T10:08:36.044+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=7544) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T10:08:36.046+0000] {standard_task_runner.py:72} INFO - Started process 7547 to run task
[2024-10-08T10:08:36.046+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_etl_01', 'run_extract_container_1', 'scheduled__2024-10-07T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmp9vji6kc5']
[2024-10-08T10:08:36.047+0000] {standard_task_runner.py:105} INFO - Job 29: Subtask run_extract_container_1
[2024-10-08T10:08:36.089+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_etl_01.run_extract_container_1 scheduled__2024-10-07T00:00:00+00:00 [running]> on host a431b2fa3f15
[2024-10-08T10:08:36.160+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='BB' AIRFLOW_CTX_DAG_ID='dag_etl_01' AIRFLOW_CTX_TASK_ID='run_extract_container_1' AIRFLOW_CTX_EXECUTION_DATE='2024-10-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-07T00:00:00+00:00'
[2024-10-08T10:08:36.161+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-08T10:08:36.181+0000] {docker.py:367} INFO - Starting docker container from image etl
[2024-10-08T10:08:36.184+0000] {docker.py:375} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-10-08T10:09:48.491+0000] {docker.py:438} INFO - End of process...
[2024-10-08T10:09:48.492+0000] {docker.py:438} INFO -     Category  ...   Like_First
[2024-10-08T10:09:48.492+0000] {docker.py:438} INFO - 0    Romance  ...  like405,348
[2024-10-08T10:09:48.492+0000] {docker.py:438} INFO - 1    Romance  ...   like70,776
[2024-10-08T10:09:48.493+0000] {docker.py:438} INFO - 2      Drama  ...   like20,087
[2024-10-08T10:09:48.493+0000] {docker.py:438} INFO - 3    Romance  ...  like219,942
[2024-10-08T10:09:48.493+0000] {docker.py:438} INFO - 4    Romance  ...  like188,131
[2024-10-08T10:09:48.493+0000] {docker.py:438} INFO - 5    Romance  ...  like127,242
[2024-10-08T10:09:48.493+0000] {docker.py:438} INFO - 6    Romance  ...   like46,338
[2024-10-08T10:09:48.494+0000] {docker.py:438} INFO - 7    Romance  ...   like57,171
[2024-10-08T10:09:48.494+0000] {docker.py:438} INFO - 8    Romance  ...  like499,627
[2024-10-08T10:09:48.494+0000] {docker.py:438} INFO - 9      Drama  ...  like127,768
[2024-10-08T10:09:48.494+0000] {docker.py:438} INFO - 10   Fantasy  ...  like260,414
[2024-10-08T10:09:48.494+0000] {docker.py:438} INFO - 11     Drama  ...   like73,380
[2024-10-08T10:09:48.494+0000] {docker.py:438} INFO - 12   Romance  ...  like176,035
[2024-10-08T10:09:48.495+0000] {docker.py:438} INFO - 13   Fantasy  ...  like161,110
[2024-10-08T10:09:48.495+0000] {docker.py:438} INFO - 14   Romance  ...  like186,938
[2024-10-08T10:09:48.495+0000] {docker.py:438} INFO - 15   Romance  ...   like30,741
[2024-10-08T10:09:48.495+0000] {docker.py:438} INFO - 16     Drama  ...   like43,133
[2024-10-08T10:09:48.495+0000] {docker.py:438} INFO - 17   Romance  ...  like247,670
[2024-10-08T10:09:48.496+0000] {docker.py:438} INFO - 18  Thriller  ...  like177,954
[2024-10-08T10:09:48.496+0000] {docker.py:438} INFO - 19   Romance  ...  like183,763
[2024-10-08T10:09:48.496+0000] {docker.py:438} INFO - 
[2024-10-08T10:09:48.496+0000] {docker.py:438} INFO - [20 rows x 9 columns]
[2024-10-08T10:09:51.784+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-08T10:09:51.785+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_etl_01, task_id=run_extract_container_1, run_id=scheduled__2024-10-07T00:00:00+00:00, execution_date=20241007T000000, start_date=20241008T100836, end_date=20241008T100951
[2024-10-08T10:09:51.831+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-08T10:09:51.854+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-08T10:09:51.855+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
